warehouse/warehouse.go

```diff
@@ -116,6 +116,7 @@ type HandleT struct {
 	destType                          string
 	warehouses                        []warehouseutils.WarehouseT
 	dbHandle                          *sql.DB
+	warehouseDBHandle                 *DB
```

Comment:
Going forward we will not be having dbHandle?

Reply:
If yes, Can we add some comments here? Since there are now 2 handles so some confusion will be there.

Reply:
Yes, going forward, the domain services need not have access to db Handle. from past experience we might get use case for having db handle but I see it getting wrapped up in warehouseDB going forward completely. I'll add a comment on the same.